datasets/biomrc/biomrc.py

```diff
@@ -0,0 +1,164 @@
+    publisher = "Association for Computational Linguistics",
+    url = "https://www.aclweb.org/anthology/2020.bionlp-1.15",
+    pages = "140--149",
+    abstract = "We introduce BIOMRC, a large-scale cloze-style biomedical MRC dataset. Care was taken to reduce noise, compared to the previous BIOREAD dataset of Pappas et al. (2018). Experiments show that simple heuristics do not perform well on the new dataset and that two neural MRC models that had been tested on BIOREAD perform much better on BIOMRC, indicating that the new dataset is indeed less noisy or at least that its task is more feasible. Non-expert human performance is also higher on the new dataset compared to BIOREAD, and biomedical experts perform even better. We also introduce a new BERT-based MRC model, the best version of which substantially outperforms all other methods tested, reaching or surpassing the accuracy of biomedical experts in some experiments. We make the new dataset available in three different sizes, also releasing our code, and providing a leaderboard.",
+}
+"""
+
+_DESCRIPTION = """\
+We introduce BIOMRC, a large-scale cloze-style biomedical MRC dataset. Care was taken to reduce noise, compared to the previous BIOREAD dataset of Pappas et al. (2018). Experiments show that simple heuristics do not perform well on the new dataset and that two neural MRC models that had been tested on BIOREAD perform much better on BIOMRC, indicating that the new dataset is indeed less noisy or at least that its task is more feasible. Non-expert human performance is also higher on the new dataset compared to BIOREAD, and biomedical experts perform even better. We also introduce a new BERT-based MRC model, the best version of which substantially outperforms all other methods tested, reaching or surpassing the accuracy of biomedical experts in some experiments. We make the new dataset available in three different sizes, also releasing our code, and providing a leaderboard.
+"""
+
+_SETTING = "A"
+_VERSION = "large"
+
+
+class BiomrcConfig(nlp.BuilderConfig):
+    """BuilderConfig for BioMRC."""
+
+    global _SETTING, _VERSION
+
+    def __init__(self, biomrc_setting="A", biomrc_version="large", **kwargs):
+        """BuilderConfig for BioMRC.
+    Args:
+      **kwargs: keyword arguments forwarded to super.
+    """
+        if biomrc_setting.lower() == "b":
+            _SETTING = "B"
+        else:
+            if biomrc_setting.lower() != "a":
+                logging.warning("Wrong Setting for BioMRC, using Setting A instead.")
+
+        if biomrc_version.lower() == "small":
+            _VERSION = "small"
+        elif biomrc_version.lower() == "tiny":
+            _VERSION = "tiny"
+        else:
+            if biomrc_version.lower() != "large":
+                logging.warning("Wrong version for BioMRC, using BioMRC Large instead.")
+
+        super(BiomrcConfig, self).__init__(**kwargs)
+
+
+class Biomrc(nlp.GeneratorBasedBuilder):
+    """BioMRC Dataset"""
+

```

Comment:
```suggestion

    BUILDER_CONFIG_CLASS = BiomrcConfig
```

Could you add this line ? This should fix the tests @PetrosStav :)

Reply:
Yes it fixed the issue, thanks! :-)